baseplate/observers/prometheus.py

```diff
@@ -74,11 +71,10 @@ def set_metrics_by_protocol(self) -> None:
         elif self.protocol == "http":
             self.metrics = PrometheusHTTPServerMetrics()
         else:
-            logger.debug(
-                "Unknown protocol %s. 'http' and 'thrift' are supported.",
+            logger.warning(
```

Comment:
For when we circle back here, I don't think that doing this by protocol is a good idea, for reasons that include:
* what about things that don't have protocols
* What if someone adds a tag for protocol that matches this but don't intend on it
* design discussion about making spans should pass in the metrics class so it doesn't assume

Reply:
I missed this one, I changed the others. Thanks for catching it.